hail/python/hailtop/utils/process.py

```diff
@@ -49,3 +50,13 @@ def sync_check_shell_output(script: str, echo=False) -> Tuple[bytes, bytes]:
 
 def sync_check_shell(script: str, echo=False) -> None:
     sync_check_shell_output(script, echo)
+
+
+def sync_check_exec(command: str, *args: str, echo: bool = False, capture_output: bool = False) -> None:
```

Comment:
What's the reason for passing `command` and `args` separately? It doesn't look like command is used by itself anywhere.